core/domain/topic_services_test.py

```diff
@@ -408,6 +411,16 @@ def test_update_subtopic_property(self):
         self.assertEqual(len(topic.subtopics), 1)
         self.assertEqual(topic.subtopics[0].title, 'Title')
 
+        # Store a dummy image in filesystem.
+        with python_utils.open_file(
+            os.path.join(feconf.TESTS_DATA_DIR, 'test_svg.svg'), 'rb',
+            encoding=None) as f:
+            raw_image = f.read()
+        fs = fs_domain.AbstractFileSystem(
+            fs_domain.GcsFileSystem(
+                feconf.ENTITY_TYPE_TOPIC, self.TOPIC_ID))
+        fs.commit('thumbnail/image.svg', raw_image, mimetype='image/svg+xml')
```

Comment:
```suggestion
        fs.commit(
            '%s/image.svg' % (constants.ASSET_TYPE_THUMBNAIL), raw_image,
            mimetype='image/svg+xml')
```

Reply:
Thanks for pointing this out, updating now.

Reply:
Done.